ui/selectors/accounts.ts

```diff
@@ -26,9 +27,11 @@ export function isSolanaAccount(account: InternalAccount) {
   return Boolean(account && account.type === DataAccount);
 }
 
-export function getInternalAccounts(state: AccountsState) {
-  return Object.values(state.metamask.internalAccounts.accounts);
-}
+export const getInternalAccounts = createSelector(
```

Comment:
Very nice work with the createSelector here!

Reply:
(future) - can we add an equality check to prove that this is not creating a new array, and causing extra rerenders?
```
it('returns same result with exact same state', () => {
  const result1 = getInternalAccounts(state);
  const result2 = getInternalAccounts(state);
  expect(result1 === result2).toBe(true)
})
```